### DISCUSSION TITLE
Vertical CSS alignment in the webpage

### DISCUSSION THREAD

**Discussion by Ayush-Singh-Git** [community/community#41163] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Actually, I want to have vertical text content on my webpage. So, how can I do without actually rotating the div container

**Comment by Priyanshu-Singh-1** [comment, marked as answer]
You can follow the css styling -

It can be done as the following -

writing-mode: vertical-rl;
text-orientation: mixed;

**Reply by Ayush-Singh-Git** [reply]
That worked for me!.... but I want to have some other method of CSS if there are any possibilities.
